src/coreclr/vm/jitinterface.h

```diff
@@ -705,6 +706,15 @@ class CEEJitInfo : public CEEInfo
         } CONTRACTL_END;
 
         m_CodeHeader = NULL;
+        
+        delete [] (BYTE*)m_CodeHeaderRW;
```

Comment:
Do we need to also delete it in `~CEEJitInfo` ?

Reply:
Added there based on your other feedback